bin/docs.test.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -ev
+
+# Make sure we use the correct repo root dir
+DIR=`dirname $0`
+REPO_ROOT=$DIR/..
+pushd $REPO_ROOT >/dev/null
+rm -rf sandbox/loopback.io/
+# Shadow clone is faster
+git clone --depth 1 https://github.com/strongloop/loopback.io.git sandbox/loopback.io
+npm run bootstrap
+pushd $REPO_ROOT/sandbox/loopback.io/ >/dev/null
+# The following 3 lines will be replaced with npm run build
```

Comment:
@raymondfeng I don't think `bundle install` should be a part of the `build` script in `loopback.io` since the script can be used there ... and on local, it's a one time install process. If you agree, I already have the changes ready to land for `loopback.io` after which I can update this script.